docs/sinco/v4.x/2_tutorials/5_restricting_permissions.md

```diff
@@ -47,8 +47,7 @@ $ deno test --allow-read="C:\Program Files\Google\Chrome\Application\chrome.exe,
 
 ## `--allow-run`
 
-This permission is required because Sinco runs a subprocess for Chrome or
-Firefox, to start a headless browser instance. So the argument to pass to this
+This permission is required because Sinco runs a subprocess for Chrome, to start a headless browser instance. So the argument to pass to this
 would be the path to your browsers executable:
```

Comment:
should this be "browser's executable" ?

Reply:
hmm i think if we getting into the grips of it, i dont think so?

like `browser's executable` is like saying "browser es executable", whilst i guess the executable word belongs to the "browser", it dont sound right? idk lol... BRENO where u at my fam